comptests/motr/test_motr_k8s_sanity.py

```diff
@@ -251,9 +233,28 @@ def test_cluster_shutdown_with_m0crate(self):
         config_file = os.path.join(os.getcwd(), "config/motr/test_29707_m0crate_workload.yaml")
         remote_file = TEMP_PATH + config_file.split("/")[-1]
         for node in self.motr_obj.get_node_pod_dict():
-            self.update_m0crate_config(config_file, node)
+            self.motr_obj.update_m0crate_config(config_file, node)
             self.motr_obj.m0crate_run(config_file, remote_file, node)
         self.motr_obj.shutdown_cluster()
         for node in self.motr_obj.get_node_pod_dict():
-            self.update_m0crate_config(config_file, node)
+            self.motr_obj.update_m0crate_config(config_file, node)
             self.motr_obj.m0crate_run(config_file, remote_file, node)
+
+    @pytest.mark.tags("TEST-29706", "TEST-29709")
```

Comment:
The test framework ( corbot) does not support a test with multiple tags.

Reply:
I have added another test for the second TEST ID.